SimonImanigirimpuhwe/my-brand-website
# #44 ft(blog): fetch all blogs from DB

#### What does this PR do?

- Fetch all blogs from DB

#### Description of Task to be completed?

- Query all blogs in DB

- Manipulate the DOM to display the all blogs

#### How should this be manually tested?

- Clone the repo

- In the directory cd this branch ` ft-query-all-articles-#44`

- In your code editor launch live server 

- Then in your browser go to localhost and navigate to the blog page you should see displayed blogs from DB

- If you have an admin account login and get redirected to the dashboard

- Click on `Posts` 

> You should see list of articles in DB with management options

#### Any background context you want to provide?

-To be able to see articles management options you should have an admin account

#### What are the relevant Trello stories?

- [#44](https://trello.com/c/u8w16sem/44-view-list-of-all-articles)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
